src/main.rs

```diff
@@ -120,19 +123,24 @@ fn setup(args: &[String]) -> (Session, Player) {
     matches.opt_str("password"));
     session.login(credentials).unwrap();
 
+     
+    let mixer_name = matches.opt_str("mixer").unwrap_or("SoftMixer".to_owned());
```

Comment:
`mixer::find` should take an `Option<&str>`, and take care of choosing a default rather than picking it here, similar to how the audio backend is done.